Services/CO.CDP.Organisation.WebApi.Tests/UseCase/UpdateOrganisationUseCaseTest.cs

```diff
@@ -315,10 +315,12 @@ public async Task Execute_ShouldNotInsertIdentifier_WhenIdentifierIdIsEmptyOrNul
         var organisation = OrganisationWithPponIdentifier;
         _organisationRepositoryMock.Setup(repo => repo.Find(_organisationId)).ReturnsAsync(organisation);
 
-        var result = await UseCase.Execute((_organisationId, command));
 
-        result.Should().BeTrue();
-        _organisationRepositoryMock.Verify(repo => repo.SaveAsync(organisation, AnyOnSave()), Times.Once);
+        Func<Task> act = async () => await UseCase.Execute((_organisationId, command));
+
+        await act.Should()
+           .ThrowAsync<InvalidUpdateOrganisationCommand>();         
+        
```

Comment:
I see a lot of code added to the use case, but only a single test case CHANGED. It doesn't seem like all the changes are covered by tests?

Reply:
I have moved around existing logic. and most of the logic is covered by test cases defined .